src/commands/remove.cr

```diff
@@ -0,0 +1,84 @@
+require "./command"
+require "./io"
+
+module Shards
+  module Commands
+    class Remove < Command
+      def run(url : String)
```

Comment:
Everything said above for the `add` command also applies to the `remove` command.